doc/concept/clg.md

```diff
@@ -0,0 +1,36 @@
+### CLG, Complex Logic Gate
+CLG, Complex logic gate, implements BB, Basic Behavior, when receiving input in
+form of BI, Basic Inormation. Neurons should implement CLGs. Lets take the
+following formular as granted.
+
+```
+BB = CLG(BI)
+```
+
+That means that basic behavior is created when some sort of complex logic
+operation is processed with respect to some fundamental input. In computer
+science there are logic gates implementing logic operations causing some
+action. The concept of CLGs presumes the need of more sophisticated logic gates
+to mimic human behavior. The theory is that as soon as there are enough
+sufficient CLGs intellectual explosion will happen.
+
+### BI, Basic Information
+BI, Basic Information, represent fundamental information that must be known to
+reach a higher level of intelligence. The most fundamental BIs like numbers
+from the decimal number system or letters from the latin alphabet simply need
+to be provided. A human and a machine respectivly cannot think of something
+totally unknown out of nothing. There needs to be information in the first
+place some kind of intelligence can collect and connect to generate new
+knowledge. Looking at the process of how humans learn, we have teachers and
+other humans who explain fundamentel concepts that help to explore the world by
+our own. Having that said there is a need for some guidance for a machine to
+learn.
+
+### BB, Basic Behavior
+BB, Basic Behavior, can probably be seen as instinct, intuition or talent.
+There are presets within humans. These enable us to do things without actively
+thinking about them. These things are might in our DNA, subconscious mind or as
+football trainers would point at it, in our blood. BB lets us breathe from the
+first second of our life, makes us fear dangerous things and lets us explore
+our environment using our inborn senses. So some sort of BB a machine needs to
+have to learn guided and later autonomous.
```

Comment:
This is still to abstract. The idea is nice but currently it means nearly nothing. This ideas need to flow into my mental models to drive technical details of implementation.